src/components/outlines/editor/index.js

```diff
@@ -1,27 +1,22 @@
+import React, { PropTypes } from 'react';
 import reactStamp from 'react-stamp';
+import EditorFactory from 'components/editor';
```

Comment:
But it also appears to be used by characters and elements. Should removal of this file at least come after #58 when we have a component on top of Editor that provides mentions?